### PR TITLE
Bug fix in map binding

### PR DESCRIPTION
**Pull request by hwsoderlund**
Closes issue #1285 

I came across this after upgrading from revel 0.16 to 0.19. 

NOTE: I have run the sample code posted by @acegishniz in the issue, and it seems to work as expected with this fix. 

I would very much appreciate if this could be merged to the 0.19.1 release.

### CODE REVIEW

**Comment by notzippy** [comment]
Thanks, I see no reason why this cannot be merged to master to make a 0.19.1 release. I was having problems creating a test case for this scenario - is it possible that you can create one for this ?

**Comment by hwsoderlund** [comment]
I'll take a look and see what I can come up with.

**Comment by hwsoderlund** [comment]
Updated PR with test. Use the "Hide whitespace changes" option for easier viewing.

File: results_test.go [hunk 1]
```diff
@@ -5,29 +5,29 @@
...
-	redirect:= RedirectToURLResult{fmt.Sprintf("/hotels/index/foo")}
+	redirect := RedirectToURLResult{fmt.Sprintf("/hotels/index/foo")}
 	resp := httptest.NewRecorder()
-	c := NewTestController(resp,showRequest)
-	redirect.Apply(c.Request,c.Response)
+	c := NewTestController(resp, showRequest)
+	redirect.Apply(c.Request, c.Response)
 	if resp.Header().Get("Location") != "/hotels/index/foo" {
-		t.Error("Failed to set redirect header correctly. : %s", resp.Header().Get("Location"))
+		t.Errorf("Failed to set redirect header correctly. : %s", resp.Header().Get("Location"))
```
Comment (hwsoderlund) [inline results_test.go]: NOTE: This is strictly unrelated to the current PR. It was broken, not sure why it has gone unnoticed.

**Comment by notzippy** [comment]
Great, that makes much more sense now...

**Comment by notzippy** [comment]
Sorry for the late merge, it slipped my mind

**Comment by hwsoderlund** [comment]
Ok, great. Thanks. Any chance this could also be merged into the 0.19 release?

**Comment by notzippy** [comment]
@hwsoderlund I was thinking a new tag for revel 0.19.1 - considering the other issue with the compression libraries that was failing travis

**Pull request merged**